neoload/manifest.json

```
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3d16e6da-7ac2-47b4-95c0-0d221686f05a",
+  "app_id": "neoload",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor and Analyze NeoLoad Performance Test Results",
+    "title": "NeoLoad",
+    "media": [{
+        "media_type": "image",
+        "caption": "NeoLoad Performance Testing dashboard",
+        "image_url": "/images/neoload-dashboard.png"
```

```suggestion
        "image_url": "images/neoload-dashboard.png"
```
The image path should be this